RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -1013,11 +1045,16 @@ namespace {
     int getPhiTensorIndex(const CellIndex& cellIndex) const { return cellIndex.phi + maxPhiIndex(); }
 
     bool tryGetCellIndex(double deltaEta, double deltaPhi, CellIndex& cellIndex) const {
-      static auto getCellIndex = [](double x, double maxX, double size, int& index) {
+      static auto getCellIndex = [this](double x, double maxX, double size, int& index) {
         const double absX = std::abs(x);
         if (absX > maxX)
           return false;
-        const double absIndex = std::floor(std::abs(absX / size - 0.5));
+        double absIndex;
+        if ( disable_CellIndex_workaround_ ) {
```

Comment:
this `if` should be inverted: when `disable_CellIndex_workaround_ == false` the old (DeepTau v2) grid definition should be used, which is `std::floor(std::abs(absX / size - 0.5));`.

Reply:
Ok, I inverted the definition of the disable_CellIndex_workaround_ flag in the above lines of code and changed the default to false, so that the behaviour of the old code remains the default.

Reply:
Ok, so we consider the former behaviour as the 'workaround' for being compatible with the training, right? Also fine :)

Reply:
Right, since the correct (and initially intended) definition of the grid is `std::floor(absX / size + 0.5)`, for me it makes more sense to call the former behaviour "workaround".